src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We have released a user survey earlier this year to learn about who our users were, how they use Cadence and how we could help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable to us and it helps us shape our roadmap for the future.
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace to get any guidance.
```

Comment:
Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating, or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale, and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace for guidance.